Models/BusinessLogic.cs

```diff
@@ -164,7 +164,7 @@ public async Task<List<long>> GetActiveQuizIdsForUser() {
 
     // Get quiz IDs from active quiz IDs
     // Fetch the quiz_id for each active_quiz_id.
-    public async Task<List<(long quizId, DateTime? startTime)>?> GetQuizIdsAndStartTimesByActiveQuizIds(List<long> activeQuizIds) {
+    public async Task<List<(long quizId, DateTime? startTime, string quizTitle)>?> GetQuizIdsAndStartTimesByActiveQuizIds(List<long> activeQuizIds) {
```

Comment:
Should this method be changed to just return a List<ActiveQuiz> instead of returning its individual components?